index.html

```diff
@@ -1530,9 +1529,7 @@ <h2>
               <a data-lt="PaymentRequest.shippingOption">shippingOption</a>
               will be set to the <a data-lt="PaymentShippingOption.id">id</a>
               of this option without running the <a>shipping option changed
-              algorithm</a>. Authors SHOULD NOT set <a data-lt=
```

Comment:
As noted in https://github.com/w3c/payment-request/issues/440 I don't understand this removal; it seems to be removing good information.